protocol/rpcprovider/provider_load_manager.go

```diff
@@ -37,15 +37,16 @@ func (loadManager *ProviderLoadManager) subtractRelayCall() {
 	if loadManager == nil {
 		return
 	}
-	loadManager.activeRequestsPerSecond.Add(^uint64(0))
+	loadManager.activeRequestsPerSecond.Add(uint64(0))
 }
 
 func (loadManager *ProviderLoadManager) getProviderLoad() string {
 	if loadManager == nil {
 		return ""
 	}
-
-	return strconv.FormatUint(loadManager.activeRequestsPerSecond.Load()/loadManager.rateLimitThreshold.Load(), 10)
+	activeRequests := loadManager.activeRequestsPerSecond.Load()
+	rateLimitThreshold := loadManager.rateLimitThreshold.Load()
```

Comment:
Need to check that this is not 0

Reply:
fixed in [a44f0ab](https://github.com/lavanet/lava/pull/1720/commits/a44f0abca9af27fcdadd4c139581c151cf8374a7)